analysis/priest/src/Mindgames.tsx

```diff
@@ -45,7 +45,10 @@ class Mindgames extends Analyzer {
   constructor(options: Options) {
     super(options);
 
-    this.active = this.selectedCombatant.hasCovenant(COVENANTS.VENTHYR.id);
```

Comment:
The generic priest Mindgames has a bunch of mixed concerns that really don't mesh well with the Discipline implementation